pkg/apis/serving/register.go

```diff
@@ -53,9 +53,9 @@ const (
 	// which Service they are created.
 	ServiceLabelKey = GroupName + "/service"
 
-	// ConfigurationGenerationLabelKey is the label key attached to a Revision indicating the
+	// DeprecatedConfigurationGenerationLabelKey is the label key attached to a Revision indicating the
 	// generation of the Configuration that created this revision
-	ConfigurationGenerationLabelKey = GroupName + "/configurationGeneration"
+	DeprecatedConfigurationGenerationLabelKey = GroupName + "/configurationGeneration"
```

Comment:
I think we'll want to eventually circle back to this label name, once it is safe to do so, so perhaps "deprecated" is too strong?

Reply:
`DONT_THINK_ABOUT_IT_ConfigurationGenerationLabelKey`